src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1098,6 +1098,8 @@ private bool HaveAllDependentServicesStopped(ServiceController[] dependentServic
         /// <param name="services">A list of services.</param>
         internal void RemoveNotStoppedServices(List<ServiceController> services)
         {
+            // You shall not modify a collection during enumeration.
+            // https://stackoverflow.com/questions/1582285/how-to-remove-elements-from-a-generic-list-while-iterating-over-it
```

Comment:
No need to reference StackOverflow. A simple comment is enough that we cannot use an enumerator.